src/main/target/SPRACINGF4EVO/target.h

```diff
@@ -34,22 +37,40 @@
 #define INVERTER_PIN_UART2      PB2
 
 #define USE_EXTI
-#define MPU_INT_EXTI            PC13
+#define GYRO_1_EXTI_PIN         PC13
+#ifdef USE_DUAL_GYRO
+#define GYRO_2_EXTI_PIN         PC5 // GYRO 2 / NC on prototype boards, but if it was it'd be here.
+#endif
+#define MPU_INT_EXTI
 
-#define USE_MPU_DATA_READY_SIGNAL
-#define ENSURE_MPU_DATA_READY_IS_LOW
+//#define USE_MPU_DATA_READY_SIGNAL
```

Comment:
Can probably be removed.

Reply:
removed for the EVODG target, kept for EVO target - via `#ifdef`